tests/Util/PhpServicesCreatorTest.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+/*
+ * This file is part of the Symfony MakerBundle package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\MakerBundle\Tests\Util;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Bundle\MakerBundle\Util\PhpServicesCreator;
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Loader\PhpFileLoader;
+use Symfony\Component\DependencyInjection\Loader\YamlFileLoader;
+use Symfony\Component\Finder\Finder;
+
+// For testing parameters conversion
+\define('GLOBAL_CONSTANT', 'value');
+
+class PhpServicesCreatorTest extends TestCase
+{
+    // For testing parameters conversion
+    const CONSTANT_NAME = 1;
+
+    /**
+     * @dataProvider getConfigurationFiles
+     */
+    public function testConvertionYamlServices(string $yamlFile, string $phpFile, string $nameTestCase)
+    {
+        $creator = new PhpServicesCreator();
+        $this->assertSame($phpFile, $creator->convert($yamlFile));
+        /*
+         * test if default comments and configuration are written but ressources can't be loaded.
+         * the container building is tested by the testLoadResources() above.
+         */
+        if ('services_load_ressources' !== $nameTestCase) {
```

Comment:
Is this special case needed anymore? It looks like the `testLoadResources()` method that you mention above does the same thing as below... so I can't understand why we need to avoid the `services_load_ressources` test case here.

Reply:
`
/*
         * test if default comments and configuration in services_load_ressources.yaml are successfully converted in php but
         * ressources "app\", "App\Controller" can't be loaded in this context, then services_load_ressources.yaml and
         * services_load_ressources.php building are tested with other fake resources by the testFixturesLoad() method above.
         */
`

Reply:
We talked about this recently in slack but if you have an other proposition now, i read you.